<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ticket</class>
 <widget class="QWidget" name="Ticket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tiket Berhasil Dipesan!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>561</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="label_tujuan">
      <property name="text">
       <string>Tujuan: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_tanggal">
      <property name="text">
       <string>Hari/Tanggal:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pengunjung">
      <property name="text">
       <string>Jumlah Pengunjung: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_harga">
      <property name="text">
       <string>Total Harga:</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btn_kembali">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Kembali ke menu awal</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
